app/services/hyrax/batch_ingest/batch_runner.rb

```diff
@@ -14,26 +14,30 @@ def initialize(batch: nil, ingest_type: nil, source_location: nil, submitter_ema
       end
 
       def run
-        initialize_batch
-        read
-        enqueue
+        if initialize_batch
```

Comment:
Funny I did this a slightly different way on Friday:
```
        return unless initialize_batch
        return unless read
        enqueue
```
@bkeese and I couldn't decide on which approach was most readable and clear.